### PR TITLE
Updated Swedish translation

### PR DESCRIPTION
**Pull request by Sopor**
It would be nice if the app have a language selector :)

### CODE REVIEW

File: term/src/main/res/values-sv/strings.xml [hunk 1]
```diff
@@ -1,52 +1,197 @@
...
-    <string name="dialog_title_initialcommand_preference">Inledande kommando</string>
+   <string name="application_terminal">Terminalemulator</string>
+   <string name="preferences">Inställningar</string>
+   <string name="new_window">Nytt fönster</string>
+   <string name="close_window">Stäng fönster</string>
+   <string name="window_list">Fönster</string>
+   <string name="prev_window">Föregående fönster</string>
+   <string name="next_window">Nästa fönster</string>
+   <string name="reset">Återställ terminal</string>
+   <string name="send_email">E-posta till</string>
```
Comment (tboy99) [inline term/src/main/res/values-sv/strings.xml]: Steff99@protonmail.com

**Review by kimkumsook — COMMENTED** [review]
``

**Comment by ATRescue** [comment]
Is Jack Palevich still alive?

**Comment by ATRescue** [comment]
Why does Android Terminal not work for 3 minutes after the device starts? (It shows no prompt and I am unable to type anything. Just a black viewport.)

**Comment by AKFU** [comment]
[ [ - @ @Sopor   ](url) ](url)

**Review by AKFU — APPROVED** [review]

**Comment by auss92** [comment]
term/src/main/res/values-sv/arrays.xml

**Comment by auss92** [comment]
#443

**Pull request closed without merging**